src/main/java/org/folio/linked/data/preprocessing/lccn/LccnNormalizer.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.linked.data.preprocessing.lccn;
+
+@FunctionalInterface
+public interface LccnNormalizer<T> {
```

Comment:
Do we need to overcomplicate this by adding `<T>`? LCCN is always going to be a string. So, instead of `T`, we can hardcode the type as `String` right? Or do you foresee any need to support other datatypes in future?

Reply:
Yes, I thought that in future we can normalize something other than just String. It should have been called just ```Normalizer``` but I forgot to rename it properly.